docs/application/native/guides/app-management/widget-viewer.md

```diff
@@ -0,0 +1,140 @@
+# EFL Widget Viewer
+
+
+The Widget Viewer API provides functionality that displays and manages Tizen Native widget applications.
+
+The main features of the Widget Viewer application API include:
+
+- Displaying widgets
+
+  You can create widget objects that can [display a widget](#display) application's buffer. The buffer is updated when the widget application updates its window.
+
+- Managing widget life-cycle
+
+  You can [manage the instance life-cycle](#manage_life-cycle) of a created widget application.
+
+- Retrieving widget information
+
+  You can [retrieve the information](#retrieve_information) of a created widget application.
+
+
+
+## Prerequisites
+
+To enable your application to use the Widget Viewer functionality:
+
+1.  To use the Widget Viewer API (in [mobile](../../api/mobile/latest/group__CAPI__WIDGET__VIEWER__EVAS__MODULE.html) and [wearable](../../api/wearable/latest/group__CAPI__WIDGET__VIEWER__EVAS__MODULE.html) applications), the application has to request permission by adding the following privilege to the `tizen-manifest.xml` file:
+
+    ```
+    <privileges>
+       <privilege>http://tizen.org/privilege/widget.viewer</privilege>
+    </privileges>
+    ```
+
+2.  To use the functions of the Widget Viewer API, include the `<widget_viewer_evas.h>` header file in your application:
+
+    ```
+    #include <widget_viewer_evas.h>
+    ```
+
+<a name="display"></a>
+## Display Widgets
+
+Widget application shares its window to the Widget Viewer application. To display the widget application's buffer you have to create a widget object. The widget application's buffer is displayed on the widget object. You can create the widget object as shown in the following code:
+
+1. Initialize Widget Viewer using `widget_viewer_evas_init()`. The parameter of `widget_viewer_evas_init()` is the window object of an application:
+
+   ```
+   Evas_Object *win = elm_win_add(NULL, "VIEWER", ELM_WIN_BASIC);
+   int ret = widget_viewer_evas_init(win);
+   ```
+
+2. Create a widget object using `widget_viewer_evas_add_widget()`. The parameters of `widget_viewer_evas_add_widget()` are parent Evas_Object, widget ID, content information, and widget update period. The widget ID is the widget application's app ID or class ID. Content information is the bundle encoded data, which contains extra information on the widget. You can retrieve the content information using `widget_viewer_evas_get_content_info()`:
```

Comment:
@rishu-samsung I want to get your opinion. Is the "You can retrieve the content information using `widget_viewer_evas_get_content_info()`" sentences needed here?

Reply:
As we are explaining about "content information" before that line, it is fine to have a sentence on "how to retrieve content information". 
To maintain consistency, we have to add a colon (:) at the end of the sentence before a code block. As we are explaining about `widget_viewer_evas_add_widget()` and its parameters in detail, I think it is fine to have this sentence.

Reply:
The content flow itself is not bad to me, too. 
My point was that : 
The last function mentioned, `widget_viewer_evas_get_content_info()` is described in the sample code of "Retrieve Widget Information" section. 

At here, as paragraph is about "create a widget object", only `widget_viewer_evas_add_widget()` is in the following sample code.

As the sample code is focusing on `widget_viewer_evas_add_widget()`, 
how about also focusing to the `widget_viewer_evas_add_widget()` in the description,
while focusing to the `widget_viewer_evas_get_content_info()` in the "Retrieve Widget Information"?

Reply:
OK. 
Then, we can avoid this sentence here.
Thank you for explaining.